### ISSUE TITLE
How to get PID of running browser?

### ISSUE THREAD

**Issue by serge-nikitin** [tebeka/selenium#29]
Is it possible to get PID of running browser (Firefox) as Remote WebDriver? Can not find anything about process / pid in API...

**Comment by tebeka** [comment]
I don't think this is possible. I can't see to find anything in the spec (see [here](https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol) and [here](https://w3c.github.io/webdriver/webdriver-spec.html)).

If you do find such a method in the wire API it should be easy to implement.
